core-bundle/src/Routing/UrlGenerator.php

```diff
@@ -186,13 +190,13 @@ private function addHostToContext(RequestContext $context, array $parameters, in
 
     /**
      * Extracts host and port from the domain.
-     *
-     * @return array<(string|null)>
```

Comment:
It seems wrong to remove this entirely. Why not adjust it instead?

Reply:
Because now it returns a tuple `[string, ?int]` and PHP can not annotate those, yet. :smile: 

Do you prefer `@return array<(string|int|null)>`?

Reply:
We could use psalms notation of object-like arrays `@return array{string, int|null}`, though this isn't even enforced in our current psalm level.

Reply:
> Do you prefer `@return array<(string|int|null)>`?

I do.

Reply:
Adressed in e3b80e01eb5868796b6b193829ac39756b304845.